src/PokemonGymImpl.java

```diff
@@ -177,7 +177,7 @@ public void performAttackPlayer(Pokemon pokemon, Pokemon gymPokemon, String atta
                     case "leafstorm" -> grass.leafStorm(pokemon, gymPokemon);
                     case "solarbeam" -> grass.solarBeam(pokemon, gymPokemon);
                     case "leechseed" -> grass.leechSeed(pokemon, gymPokemon);
-                    default -> grass.leaveBlade(pokemon, gymPokemon);
+                    default -> grass.leafBlade(pokemon, gymPokemon);
```

Comment:
We bedoelden hier natuurlijk de verdedigingsaanval "leaveBlade", als in "ga weg zwaard". Het was zeer zeker geen typefout... of misschien wel...